include/wlr/types/wlr_xdg_foreign_v1.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * This an unstable interface of wlroots. No guarantees are made regarding the
+ * future consistency of this API.
+ */
+#ifndef WLR_USE_UNSTABLE
+#error "Add -DWLR_USE_UNSTABLE to enable unstable wlroots features"
+#endif
+
+#ifndef WLR_TYPES_WLR_XDG_FOREIGN_V1_H
+#define WLR_TYPES_WLR_XDG_FOREIGN_V1_H
+
+#include <wayland-server-core.h>
+#include <wlr/types/wlr_xdg_foreign_registry.h>
+
+struct wlr_xdg_foreign_v1 {
+	struct wl_global *exporter_global;
+	struct wl_global *importer_global;
+
+	struct {
+		struct wl_global *global;
```

Comment:
We have `exporter_global` and `exporter.global`. Do we need both?

Reply:
Indeed, this is a mistake, will remove.